colossus-metrics/src/main/scala/colossus/metrics/collectors/Histogram.scala

```diff
@@ -104,12 +106,28 @@ object Histogram extends CollectorConfigLoader{
     * @return
     */
   def apply(address : MetricAddress, configPath : String)(implicit ns : MetricNamespace) : Histogram = {
+    addToNamespace(address, configPath, None)
+  }
+
+  /**
+    * Create a Histogram with the following address.  Source the config from the provided Config object,
+    * instead of the MetricNamespace's Config
+    *
```

Comment:
New constructor which takes in a Config object.  I modified all `Collector` constructors so that both this constructor and the other one with no Config object, funnel into the same function.